build_tools/cmake/iree_setup_toolchain.cmake

```diff
@@ -20,13 +20,25 @@ endif()
 if(IREE_USE_LINKER)
   set(IREE_LINKER_FLAG "-fuse-ld=${IREE_USE_LINKER}")
 
+  iree_append("${IREE_LINKER_FLAG}"
```

Comment:
change to `IREE_LINKER_FLAGS` since there are multiple flags in the variable now?

Reply:
No there's still only one flag. This is using the helper function above which appends the first argument to all the trailing arguments

Reply:
Oh, that name could be more clear then :P 
`iree_append_to_lists` or something, idk

your change is fine as-is

Reply:
Yeah agreed :-) Given that the implementation is like 10 lines up I wasn't too stressed about it though